data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApi.java

```diff
@@ -163,6 +163,10 @@ private void addValidatorHandlers(DataProvider dataProvider) {
     ChainDataProvider provider = dataProvider.getChainDataProvider();
     app.get(GetAttestation.ROUTE, new GetAttestation(provider, jsonProvider));
     app.get(GetValidators.ROUTE, new GetValidators(provider, jsonProvider));
+    app.get(
+        tech.pegasys.artemis.beaconrestapi.handlers.validator.GetBlock.ROUTE,
```

Comment:
this should be imported?

Reply:
oh it's a name collision

Reply:
yeh i'd have to rename it... could possibly call it GetNewBlock to avoid the name collision...